Runtime/OperatingSystem/FilesBrowser.cs

```diff
@@ -0,0 +1,64 @@
+using UnityEngine;
+
+namespace StansAssets.Foundation.OperatingSystem
+{
+    /// <summary>
+    /// System File browser access point.
+    /// </summary>
+    public static class FilesBrowser
+    {
+        /// <summary>
+        /// Open new file browser window at given path.
+        /// </summary>
+        /// <param name="path">absolute system path.</param>
+        public static void OpenAtPath(string path)
+        {
+            switch (Application.platform)
+            {
+                case RuntimePlatform.WindowsEditor:
+                case RuntimePlatform.WindowsPlayer:
+                    OpenAtPathMacOs(path);
+                    break;
+                case RuntimePlatform.OSXEditor:
+                case RuntimePlatform.OSXPlayer:
+                    OpenAtPathWindows(path);
+                    break;
```

Comment:
do we need 'default:' case here with something like Debug.Log("this platform is not supported") message?
just in case I left this code for, let's say, LinuxEditor platform.
if you think this Debug.Log is redundant here, please, leave this code like it is

Reply:
Good catch.
I think `PlatformNotSupportedException` would work better here.